packages/rxjs-firebase/src/operators/__tests__/waitForData.test.ts

```
@@ -0,0 +1,34 @@
+import { type DocumentSnapshot } from '@firebase/firestore'
+import { from, of } from 'rxjs'
```

ESLint

[error] Unable to resolve path to module 'rxjs'.

(import-x/no-unresolved)